features/all-positions/LiquidationActionDialog.tsx

```diff
@@ -133,7 +133,7 @@ const LiquidationActionDialog = (props: DialogProps) => {
 
     const liquidationTimeRemaining =
       Number(liquidatedPosition.liquidationTimestamp) +
-      Number(withdrawalLiveness) -
+      Number(liquidationLiveness) -
```

Comment:
Ah good catch, but why would this have been an error if withdrawal liveness and liquidation liveness are the same value?

Reply:
would not have been an issue if they were the same.